### DISCUSSION TITLE
Refresh Host Capabilities via Ansible Module

### DISCUSSION THREAD

**Discussion by ombre8** [oVirt/ovirt-ansible-collection#672] [Q&A] [answered]
Hi there,
we manage our VMS on oVirt via Ansible. One of those VMs should have an USB-Disk attached, which we have to change from time to time. Most of the time everything works just fine, with powering down the VM, run Ansible and find the Host-Device attached to the VM. Sometimes we do have to click the "Refresh Capabilities" Button in the GUI. Is there a way to trigger this via Ansible?
I found [this API Endpoint](https://ovirt.github.io/ovirt-engine-api-model/master/#services/host/methods/refresh) but couldn't find something corresponding in Ansible.

Thanks for your Help

**Comment by mnecas** [comment, marked as answer]
Hi, posted https://github.com/oVirt/ovirt-ansible-collection/pull/673 
hope it will help

**Reply by ombre8** [reply]
nice, thank you
